pkg/chains/formats/slsa/internal/material/material_test.go

```diff
@@ -371,7 +371,9 @@ func TestAddStepImagesToMaterials(t *testing.T) {
 	}}
 	for _, tc := range tests {
 		var mat []common.ProvenanceMaterial
-		if err := AddStepImagesToMaterials(tc.steps, &mat); err != nil {
+		var err error
+		mat, err = FromStepImages(tc.steps)
+		if err != nil {
```

Comment:
same here, using https://pkg.go.dev/errors@master#Is

Reply:
Would recommend doing as a follow up!